README.md

```diff
@@ -27,6 +27,8 @@ $ ./csp_collector
 - `POST /`: accepts a CSP violation report (recommended to use `/csp` for future proofing though).
 - `POST /csp`: accepts a CSP violation report.
 - `POST /csp/report-only`: same as `/csp` but appends a `report-only` attribute to the log line. Helpful if you have enforced and report only violations and wish to separate them.
+- `OPTIONS /report/csp`: CORS implementation for the Reporting-API.
```

Comment:
instead of `/report/csp` let's use `/reporting-api/csp` to make it explicit what it is for (and avoid confusion with the existing `/csp/report` endpoint).

Reply:
agreed